log.js

```diff
@@ -209,11 +209,11 @@ let getAppenderForLevel = function(){
     if(appenderForLevel === null) {
         appenderForLevel = () => null;
 
-        try {
-            appenderForLevel = require('log_appenders').forLevel;
-        } catch(e) {
-            new Logger("log", () => null).debug("No appenders found for log", e);
-        }
+        // try {
```

Comment:
Hi @jpg0 I commented this out as i'm not sure where log_appenders is coming from, is this needed?

Reply:
The original use-case here was to allow custom log appenders for JS code. Specifically, in my case, to log errors to Telegram. I didn't want to require boilerplate in each JS script, so I put it into the log library, and had it "optimistically" look up a custom module to search for extra appenders. In my installation I provided an appender to log to Telegram.

I don't believe that anyone else has used this functionality though, so I would say that it's fine to remove.